MekHQ/src/mekhq/campaign/personnel/familyTree/Genealogy.java

```diff
@@ -264,9 +257,9 @@ public boolean checkMutualAncestors(Person person, Campaign campaign) {
             // If so, decrease remaining search depth
             depth--;
             // Then parse through the parents
-            for (UUID parent : getParents()) {
+            for (Person parent : getParents()) {
                 // And add all of their returned ancestors to the list
-                ancestors.addAll(campaign.getPerson(parent).getGenealogy().getAncestors(depth, campaign));
+                ancestors.addAll(parent.getGenealogy().getAncestors(depth));
```

Comment:
Is a call to getGenealogy() guaranteed to return non-null?

Reply:
It is. Anything that is nullable is marked as such.